modules/@apostrophecms/notification/ui/apos/components/AposNotification.vue

```diff
@@ -127,7 +133,10 @@ export default {
         console.error('Unable to find notification job:', this.notification.jobId);
         this.job = null;
       }
+    }
 
+    if (this.notification.event?.name) {
+      apos.bus.$emit(this.notification.event.name, this.notification.event.data);
```

Comment:
This has to be preceded by an API call to remove the `event` property from the notification, so that it doesn't fire more than once as the user navigates from page to page, etc.